contents/blog/5-ways-to-improve-analytics-data.md

```diff
@@ -0,0 +1,102 @@
+---
+date: 2023-09-06
+title: 5 ways to improve your analytics data
```

Comment:
Other titles I considered:
- How to build a good product analytics foundation
- How to log analytics you can trust
- 5 quick wins to log better data

Reply:
Thanks! Will give it some thought.